cognite/client/_version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.34.0"
+__version__ = "2.34.1"
```

Comment:
Extending the feature set we should bump the minor version.
```suggestion
__version__ = "2.35.0"
```